src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -44,6 +44,7 @@
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
 	public static final String AUDIO_TRANSCODE = "audio/transcode";
 	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AAC_TYPEMIME = "audio/mp4";
```

Comment:
I agree that the DLNA MIME type should be the default, but I can't see that there is any profile for AAC. All the audio profiles that I can find that starts with "AAC" is listed under "Audio Class: MPEG-4 profiles".

```audio/mp4``` is the correct mimetype for M4A/MP4/MPEG-4 audio, but not for raw AAC audio. This is why I'm saying that you should stick to MP4/M4A, and don't mix AAC into this.